backend/src/main/java/com/google/minicrm/data/Lead.java

```diff
@@ -153,9 +167,39 @@ public Entity asEntity() {
     leadEntity.setProperty("creativeId", creativeId);
     leadEntity.setProperty("notes", notes);
     leadEntity.setProperty("status", status.ordinal());
+    List<String> locationInfos = new ArrayList<String>();
```

Comment:
"new ArrayList<>" is sufficient, don't need the String type on RHS since it's inferred.

Reply:
will fix